src/DocumentationBrowserViewExtension/PackageDocumentationManager.cs

```diff
@@ -39,17 +46,64 @@ public static PackageDocumentationManager Instance
             }
         }
 
+        internal Action<ILogMessage> MessageLogged;
+
+        /// <summary>
+        /// Uses the path manager to set the fallback_doc path for DynamoCore and any Host running
+        /// </summary>
+        /// <param name="pathManager"></param>
+        internal void AddDynamoPaths(IPathManager pathManager)
+        {
+            if (pathManager is null)
+            {
+                return;
+            }
+
+            if (!string.IsNullOrEmpty(pathManager.DynamoCoreDirectory))
+            {
+                var coreDir = new DirectoryInfo(Path.Combine(pathManager.DynamoCoreDirectory, FALLBACK_DOC_DIRECTORY_NAME));
+                dynamoCoreFallbackDocPath = coreDir.Exists ? coreDir : null;
+            }
+
+            if (!string.IsNullOrEmpty(pathManager.HostApplicationDirectory))
+            {
+                var hostDir = new DirectoryInfo(Path.Combine(pathManager.HostApplicationDirectory, FALLBACK_DOC_DIRECTORY_NAME));
+                hostDynamoFallbackDocPath = hostDir.Exists ? hostDir : null;
+            }   
+        }
+
         private PackageDocumentationManager() { }
 
         /// <summary>
         /// Retrieves the markdown node documentation file associated with the input node namespace
         /// </summary>
         /// <param name="nodeNamespace">Namespace of the node to lookup documentation for</param>
         /// <returns></returns>
-        public string GetAnnotationDoc(string nodeNamespace)
+        public string GetAnnotationDoc(string nodeNamespace, string packageName)
```

Comment:
yes @sm6srw, I had the same reaction initially, https://github.com/DynamoDS/Dynamo/pull/11725/files#r646888243 - but it's actually inside an extension, I think it's okay to break this with chance of risk and if possible it would be nice to codify extensions being part of the API boundary or not but it will take a cross team effort / discussion I think.

in this case I'm okay with the change unless there are objections.